metadata.json

```diff
@@ -43,37 +42,6 @@
         "7",
         "8"
       ]
-    },
-    {
-      "operatingsystem": "OracleLinux",
```

Comment:
please only purge the operating systems that are EoL. OracleLinux should be fine, I don't know about OpenSUSE 42. FreeBSD 12 and 13 are fine, Arch is fine, OpenBSD 7 is fine as well.

Reply:
But how could this module support OSes that are not supported by augeasproviders_core?